### PR TITLE
Import CQL Libraries - Measure Composer

### PR DESCRIPTION
**Pull request by adongare**
MAT-438 MAT - Import CQL Libraries - CQL Composer
MAT-438 MAT - Import CQL Libraries - CQL Composer
MAT-440 Update Help Text

### CODE REVIEW

File: src/main/java/mat/server/FeatureFlagServiceImpl.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
-@Service
+@Service("featureFlagService")
```
Comment (serhii-ilin) [inline src/main/java/mat/server/FeatureFlagServiceImpl.java]: why do we need a name? just curious. it's usually not required

File: src/main/java/mat/server/CQLServiceImpl.java [hunk 2]
```diff
@@ -102,18 +103,21 @@
...
+
+	@Autowired
 	private MeasurePackageService measurePackageService;
-	
+
 	@Autowired
 	private UserDAO userDAO;
 
+	@Autowired
+	private FeatureFlagServiceImpl featureFlagService;
```
Comment (serhii-ilin) [inline src/main/java/mat/server/CQLServiceImpl.java]: why do you inject am Impl and not a corresponding interface?

File: src/main/java/mat/server/FeatureFlagServiceImpl.java (hunk 1 above)
Comment (adongare) [reply src/main/java/mat/server/FeatureFlagServiceImpl.java]: we don't need this. was experimenting with something else. this service is registered twice with spring container. Need refactor.

File: src/main/java/mat/server/CQLServiceImpl.java (hunk 2 above)
Comment (adongare) [reply src/main/java/mat/server/CQLServiceImpl.java]: this is because the @Service is on impl and not on interface i guess. interface did not work

Comment (serhii-ilin) [reply src/main/java/mat/server/CQLServiceImpl.java]: that should work in any case on interface. regardless of where the annotation is placed.
if it's not, then something is broken in our spring jar dependencies

**Comment by serhii-ilin** [comment]
it has merge conflicts now

**Comment by adongare** [comment]
> it has merge conflicts now

Resolved all the conflicts.

File: pom.xml [hunk 3]
```diff
@@ -605,6 +605,10 @@
                     <argLine>
                         --illegal-access=permit
                     </argLine>
+                    <excludes>
+                        <!--run DAO tests manually -->
```
Comment (serhii-ilin) [inline pom.xml]: Probably we should come up with a better filter for the unit tests like that sometime later. But this is good for now.

Comment (adongare) [reply pom.xml]: Yup, Agreed

**Pull request merged**